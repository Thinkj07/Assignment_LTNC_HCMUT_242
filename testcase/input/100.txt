INSERT a3 number
ASSIGN a3 ASSIGN