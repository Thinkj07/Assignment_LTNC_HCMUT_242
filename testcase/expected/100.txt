Invalid: ASSIGN a3 ASSIGN